climpred/comparisons.py

```diff
@@ -1,9 +1,21 @@
+import dask
 import numpy as np
 import xarray as xr
 
 from .checks import has_dims, has_min_len
 from .exceptions import DimensionError
 
+# import error
+# from .utils import _transpose_and_rechunk_to
```

Comment:
Is this fixed with circular import updates? Can the below function be moved back to utils?